docs/components/input/README.md

```diff
@@ -195,29 +195,7 @@
             ],
             "events": [
               {
-                "name": "input",
-                "arguments": "value, event",
-                "description": "Input event is emitted while typing with the current input value.",
-              },
-              {
-                "name": "blur",
-                "arguments": "event",
-                "description": "Blur event is emitted when input loses focus.",
-              },
-              {
-                "name": "focus",
-                "arguments": "event",
-                "description": "Focus event is emitted when input gains focus.",
-              },
-              {
-                "name": "paste",
-                "arguments": "event",
-                "description": "Paste event is emitted when text is pasted into input.",
-              },
-              {
-                "name": "keydown",
-                "arguments": "event",
-                "description": "Keydown event is emitted when a key is pressed.",
+                "description": "All event listeners are passed through to/from the component."
```

Comment:
Should we clarify that this passes events through to the input element? There are many other elements like the label that are part of that component so it isn't as simple as something like button. It's probably kind of intuitive that it works that way, but I think we might be better off being explicit anyway